cluster-provision/k8s/1.17/provision.sh

```diff
@@ -322,6 +322,9 @@ docker pull quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
 cp -rf /tmp/cnao/ /opt/
 for i in $(grep -A 2 "IMAGE" /opt/cnao/operator.yaml |grep value | awk '{print $2}'); do docker pull $i; done
 
+# Pre pull local-volume-provisioner
+grep -A 2 "IMAGE" /tmp/local-volume.yaml | grep value | awk '{print $2}' | xargs docker pull
```

Comment:
Is this needed in 1.18 too?

Reply:
yep, mentioned it in the PR, 
unless 1.18 doesnt have local-volume-provisioner,
i will verify,
question out of the scope of the PR, is it fine that we dont have common.sh for k8s ?
and every change we need to duplicate ? 
its prune to errors.

Reply:
updated 1.18, its needed as well
if it looks good i will updated hashes, thanks